ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -300,20 +300,37 @@
       (= (:gtfs/route-hash-id-type route-hash-id-type) "short-long-headsign")
       (= (:gtfs/route-hash-id-type route-hash-id-type) "long-headsign")))
 
+(def selected-change-keys #{:removed-trips :trip-stop-sequence-changes-lower
```

Comment:
Alternative, any better?
```(defn- list-route-changes-with-same-route-hash-id [all-changes single-change]
  ;; Filter nil values
  (keep
    (fn [c]
      (let [x (select-keys c selected-change-keys)
            x (if (= :removed (:change-type x))
                ;; Remove trip and stop changes from route summary if route has change-type :removed
                (dissoc x :trip-stop-sequence-changes-lower
                        :trip-stop-time-changes-lower
                        :trip-stop-sequence-changes-upper
                        :trip-stop-time-changes-upper
                        :removed-trips)
                x)]
        ;; Return nil because keep won't work with false values.
        (when (= (:route-hash-id x) (:route-hash-id single-change))
          (if (= :removed (:change-type x))
                ;; Remove trip and stop changes from route summary if route has change-type :removed
                (dissoc x :trip-stop-sequence-changes-lower
                        :trip-stop-time-changes-lower
                        :trip-stop-sequence-changes-upper
                        :trip-stop-time-changes-upper
                        :removed-trips)
                x))
        ))
    all-changes))```